tqdm.py

```diff
@@ -0,0 +1,64 @@
+import time
+from inspect import isgenerator
+
+PROGRESS_BAR_LEN = 20
+
+
+class Tqdm(object):
+    """
+    Iterable that prints status of other iterable objects
```

Comment:
You are writing a package that will serve other developers, try to think of what information you would want to know when using this external package.